kernel/kernel-api/src/test/scala/io/delta/kernel/internal/SnapshotManagerSuite.scala

```diff
@@ -744,12 +735,6 @@ class SnapshotManagerSuite extends AnyFunSuite with MockFileSystemClientUtils {
       versionToLoad = Optional.of(17),
       expectedErrorMessageContains = "missing log file for version 0"
     )
-    testExpectedError[InvalidTableException](
-      deltaFileStatuses(15L until 25L) ++ singularCheckpointFileStatuses(Seq(20L)),
-      startCheckpoint = Optional.of(20),
```

Comment:
this just throws an IllegalArgumentException when we invoke the log listing util ... which is expected.